src/main/java/persistence/sql/ddl/Entity.java

```diff
@@ -0,0 +1,80 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import persistence.sql.ddl.exception.IncorrectIdFieldException;
+import persistence.sql.ddl.exception.NotEntityException;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+
+public class Entity {
+    private String name;
+
+    private EntityIdField idField;
+
+    private List<EntityField> fields;
```

Comment:
```suggestion
    private final String name;

    private final EntityIdField idField;

    private final List<EntityField> fields;
```
불변하게 설정하면 좀 더 안정적인 객체가 될 것 같아요. 다른 클래스의 변수나 상수도 한번 검토해주세요 :)